modules/cnf-performing-end-to-end-tests-disconnected-mode.adoc

```diff
@@ -38,9 +38,8 @@ where:
 [source,terminal,subs="attributes+"]
 ----
 $ podman run -v $(pwd)/:/kubeconfig:Z -e KUBECONFIG=/kubeconfig/kubeconfig \
--e DISCOVERY_MODE=true -e FEATURES=performance -e IMAGE_REGISTRY="<disconnected_registry>" \
--e CNF_TESTS_IMAGE="cnf-tests-rhel8:v{product-version}" \
-/usr/bin/test-run.sh -ginkgo.focus="\[performance\]\ Latency\ Test"
+-e IMAGE_REGISTRY="<disconnected_registry>" -e CNF_TESTS_IMAGE="cnf-tests-rhel8:v{product-version}" \
+/usr/bin/test-run.sh 
```

Comment:
before specifying the entry point you should specify the image the podman need to run. in this case it should be a combination of ```<disconnected_registry>/cnf-tests-rhel8:v{product-version}```.  otherwise the command will hit ````Error: invalid reference format````

Reply:
check this as an example how there is an image URL before the entry point:
https://github.com/openshift/openshift-docs/pull/69462/files#diff-6531bf10f5dca593e3125fae3c7e3e054dfd7ee879e18725df3287bf5225339aR24

Reply:
updated. Thanks.